client/dashboard/profile-wizard/steps/store-details.js

```diff
@@ -117,7 +118,36 @@ class StoreDetails extends Component {
 			},
 		} );
 
-		await updateProfileItems( { setup_client: values.isClient } );
+		const profileItemsToUpdate = { setup_client: values.isClient };
+		const cbdSlug = 'cbd-other-hemp-derived-products';
+		const hasCbdIndustry =
+			profileItems.industry &&
+			profileItems.industry.length &&
+			profileItems.industry.some( ( industry ) => {
+				return cbdSlug === industry || cbdSlug === industry.slug;
+			} );
```

Comment:
Nitpick: since this block already iterates all of the `profileItems.industry` array, it's not offering any performance benefit in the `if()` statement below. We could just always filter the `profileItems.industry` array for non-US `region`s.

Reply:
Good call 😄